toolbox/bulk_processing/validators.py

```diff
@@ -123,3 +125,23 @@ def validate(expected_capacity, **kwargs):
                 f'CE Unit Expected Capacity "{expected_capacity}" cannot be null, blank or zero')
 
     return validate
+
+
+def ce_e_has_expected_capacity():
+    def validate(expected_capacity, **kwargs):
+        if kwargs['row']['ADDRESS_TYPE'] == 'CE' and kwargs['row']['ADDRESS_LEVEL'] == 'E' and (
+                kwargs['row']['TREATMENT_CODE'] not in {'CE_LDCEE', 'CE_LDCEW'}) and (
+                not expected_capacity.isdigit() or int(expected_capacity) == 0):
+            raise Invalid(
+                f'Expected Capacity "{expected_capacity}" must be greater than 0')
```

Comment:
Worth feeding back this is a CE E case with a treatment code we expect an expected capacity for, it's not going to be easy to look up rows in the massive file.